src/store/modules/Content/mutations.ts

```diff
@@ -0,0 +1,65 @@
+import { MutationTypes } from "./mutation-types"
+import { MutationTree } from "vuex"
+import { ContentState } from "./index"
+import { ButtonModel, Field, GetContentMediakiwiResponseModel, Grid, PageItem } from "@/models/Mediakiwi/Response/Content/GetContentMediakiwiResponseModel"
+import { mediakiwiLogic } from "@/utils/mediakiwiLogic"
+import PageModel from "@/models/PageModel"
+import ResourceModel from "@/models/Mediakiwi/ResourceModel"
+import { PostContentMediakiwiResponseModel } from "@/models/Mediakiwi/Response/Content/PostContentMediakiwiResponseModel"
+
+export type Mutations<S = ContentState> = {
+  [MutationTypes.SET_CONTENT](state: S, payload: GetContentMediakiwiResponseModel): void,
+  [MutationTypes.SET_GRIDS](state: S, payload: Grid[]): void,
+  [MutationTypes.SET_FOLDERS](state: S, payload: Field[]): void,
+  [MutationTypes.SET_BUTTONS](state: S, payload: ButtonModel[]): void,
+  [MutationTypes.SET_PAGE](state: S, payload: PageModel): void,
+  [MutationTypes.SET_FIELDS](state: S, payload: Field[]): void,
+  [MutationTypes.SET_RESOURCES](state: S, payload: ResourceModel[]): void,
+  [MutationTypes.SET_POST_CONTENT](state: S, payload: PostContentMediakiwiResponseModel|any): void,
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8560396)
```suggestion
  [MutationTypes.SET_POST_CONTENT](state: S, payload: PostContentMediakiwiResponseModel|unknown): void,
```